### ISSUE TITLE
Consistent Program Code

### ISSUE THREAD

**Issue by Simon-Deuring** [Simon-Deuring/labgrid-frontend-mle#72]
**Description:**

When working fast developers risk that the program code gets messy. To accomplish consistent and clean code a linter can be used. Especially for the frontend such a linter shall be added. Angular does not include this functionality but allows adding packages for this purpose.

**Acceptance criteria:**
- [x] Add a package for linting to the dev dependencies
- [x] Run the linting command for the Angular code
- [x] The README.md is updated and states that the new command shall be runned before a new pull request is created

**Additional information:**
Before starting have a look at the Angular documentation about linting: https://angular.io/cli/lint
